var/spack/repos/builtin/packages/ruby-open4/package.py

```diff
@@ -0,0 +1,41 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+#
+
+from spack import *
+
+
+class RubyOpen4(RubyPackage):
+    """Ruby open child process with handles on pid, stdin, stdout, and stderr:
+    manage child processes and their io handles easily."""
+
+
+    homepage = "https://rubygems.org/gems/open4"
+    url      = "https://rubygems.org/downloads/open4-1.3.4.gem"
+
+    version('1.3.4', '24c0bb0abdbf098ad116c1bb5a6a1905', expand=False)
+
+    def install(self, spec, prefix):
+        gem('install', '--local', 'open4-{0}.gem'.format(self.version))
```

Comment:
Does this install *into* the interpreter directory?  If so, we should change it work like Python does, with activation or addition to `RUBYLIB`.  Does `gem install` take a `--prefix` argument?

Reply:
gem install takes --install-dir